docs/src/main/sphinx/release/release-422.md

```diff
@@ -0,0 +1,49 @@
+# Release 422 (12 Jul 2023)
+
+## General
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18005`)
+* Fix queries hanging when the `task.retry-policy` configuration property is set
+  to `TASK` and other worker node failures occur. ({issue}`18175 `)
+
+## Security
+
+* Add OAuth 2.0 support for validating JWT types. ({issue}`17640`)
```

Comment:
Sorry.. not sure what this means .. maybe

Add support for validating JWT types with OAuth 2.0 authentication